configure

```diff
@@ -572,12 +573,13 @@ pass_config_kernel_registries()
 
 		# If we encounter a slash, it means the name of the configuration
 		# and the kernel set needed by that configuration are different.
-		if [[ "${list}" == *[/]* ]]; then
+		if [[ ${list} = */* ]]; then
 
 			#echo "Slash found."
 			klist=""
 			clist=""
-			for item in "${list}"; do
+
+			for item in ${list}; do
 
```

Comment:
This is a bug -- single quotes makes the loop only execute once, even if `${list}` contains space-separated items.

If the items in `${list}` may need to contain spaces, then declaring `${list}` a Bash array variable and using it such as with `"${list[@]}"` will work.

Reply:
Agreed, this was a bug. I think we might have been lucky in that the `config_registry` didn't make use of syntax that would have needed the loop.